util/zenfs.cc

```diff
@@ -443,12 +460,37 @@ IOStatus zenfs_tool_copy_dir(FileSystem *f_fs, const std::string &f_dir,
 
   return s;
 }
+IOStatus zenfs_create_directories(FileSystem *fs, std::string path) {
+  std::string dir_name;
+  IODebugContext dbg;
+  IOOptions opts;
+  IOStatus s;
+  int p = 0;
+
+  if (path.back() != '/') path += '/';
+
+  while ((p = path.find_first_of('/', p)) != std::string::npos) {
```

Comment:
I think `p` should be declared as `std::size_t`.
Otherwise,
```
/home/yura/ws/percona-server/storage/rocksdb/rocksdb_plugins/zenfs/util/zenfs.cc:472:43: warning: comparison of integer expressions of different signedness: ‘int’ and ‘const size_type’ {aka ‘const long unsigned int’} [-Wsign-compare]
  472 |   while ((p = path.find_first_of('/', p)) != std::string::npos) {
      |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~
```

Reply:
fixed